<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>role_createEdt</class>
 <widget class="QWidget" name="role_createEdt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>创建分身</string>
  </property>
  <widget class="QLineEdit" name="edit_name">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>8</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>30</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>名字：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>80</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>性别：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_male">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>39</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>男</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_female">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>39</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>女</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>130</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>职业：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_voc">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>未知职业</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>战士</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>法师</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>道士</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>已锁定，不可选择</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>100</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>创建</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_quit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>100</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>我还没想好</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>    注意：创建本职业分身需要消耗金币50万，非本职业分身需要150万。</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mirror.qrc"/>
 </resources>
 <connections/>
</ui>
